shared/locales/intl/unitformat.en-US.properties

```diff
@@ -37,3 +37,43 @@ duration-month-narrow[two]    = {{ value }}mths
 duration-month-narrow[few]    = {{ value }}mths
 duration-month-narrow[many]   = {{ value }}mths
 duration-month-narrow[other]  = {{ value }}mths
+
+digital-byte-short            = {[ plural(value) ]}
```

Comment:
Let's use what CLDR uses in http://unicode.org/cldr/trac/browser/trunk/common/main/en.xml and https://github.com/unicode-cldr/cldr-units-full/blob/master/main/en/units.json, i.e. `mass-ton`, `energy-joule`, `power-kilowatt`.